### ISSUE TITLE
Error: error validating monitor: 502 Bad Gateway

### ISSUE THREAD

**Issue by fatbasstard** [DataDog/terraform-provider-datadog#782]
Hi there,

Thank you for opening an issue. Please note that we try to keep the Terraform issue tracker reserved for bug reports and feature requests. For general usage questions, please see: https://www.terraform.io/community.html.

### Terraform Version

Terraform: 0.14.2
DataDog provider: 2.18.1

### Affected Resource(s)

Please list the resources as a list, for example:

-   datadog_monitor

### Actual Behavior

At random times we get a `Error: error validating monitor: 502 Bad Gateway` error when running Terraform Plan

```
Error: error validating monitor: 502 Bad Gateway: {"status":"error","code":502,"errors":["Bad Gateway"],"statuspage":"http://status.datadoghq.com","twitter":"http://twitter.com/datadogops","email":"support@datadoghq.com"}

  on .terraform/modules/xxx.monitors/main.tf line 6, in resource "datadog_monitor" "default":
   6: resource "datadog_monitor" "default" {
```

### References

Are there any other GitHub issues (open or closed) or Pull Requests that should be linked here? For example:

-   See comment in possibly related issue: 

https://github.com/DataDog/terraform-provider-datadog/issues/190#issuecomment-665208983

Rerun the plan works usually, as suggested, a retry mechanism would be nice.

**Comment by phillip-dd** [comment]
This was added for monitors here: https://github.com/DataDog/terraform-provider-datadog/pull/816/

**Comment by therve** [comment]
Hi, I'm considering this fixed by #816 , let us know if that doesn't work. Thanks!
